src/pages/SearchPage.js

```diff
@@ -101,17 +105,37 @@ class SearchPage extends Component {
      * @returns {Array}
      */
     getSections() {
+        return this.filterAdapter('');
+    }
+
+
+    /**
+     * Returns the sections needed for the OptionsSelector
+     * @param {String} searchValue
+     * @returns {Array}
+     */
+    filterAdapter(searchValue) {
```

Comment:
Confused at why this is called filterAdapter

Reply:
Regarding this, honestly, I was also confused because I am not sure which type of design pattern is preferred in this project.
I renamed it to searchSections.
But, if it is not good, can you please recommend one?